bundles/org.openhab.binding.nikohomecontrol/src/main/java/org/openhab/binding/nikohomecontrol/internal/handler/NikoHomeControlEnergyMeterHandler.java

```diff
@@ -132,13 +137,14 @@ public void dispose() {
         }
     }
 
-    private void updateProperties() {
+    private void updateProperties(NhcEnergyMeter nhcEnergyMeter) {
         Map<String, String> properties = new HashMap<>();
 
         if (nhcEnergyMeter instanceof NhcEnergyMeter2) {
             NhcEnergyMeter2 energyMeter = (NhcEnergyMeter2) nhcEnergyMeter;
-            properties.put("model", energyMeter.getModel());
-            properties.put("technology", energyMeter.getTechnology());
+            properties.put("deviceType", energyMeter.getDeviceType());
+            properties.put("deviceTechnology", energyMeter.getDeviceTechnology());
+            properties.put("deviceModel", energyMeter.getDeviceModel());
```

Comment:
Would it make sense to use `Thing.PROPERTY_MODEL_ID` ("modelId") here? In any case, consider also using constants to avoid redundancy/mistakes as the same properties are also provided in `NikoHomeControlThermostatHandler`.

Reply:
@jlaur I made these properties constants, but would prefer to stick with the names as defined. They match names in the API Niko Home Control configuration software, and make it easier to guide users.